perfkitbenchmarker/package_managers.py

```diff
@@ -205,11 +233,31 @@ def GetServiceName(self, package_name):
     package = packages.PACKAGES[package_name]
     return package.YumGetServiceName(self)
 
+  def SetupProxy(self):
+    """
+        This function setup proxy if your cloud is currently installed behind proxy server
+    """
+    super(YumMixin, self).SetupProxy()
+    yum_proxy_file = "/etc/yum.conf"
+
+    commands = []
+
+    add_proxy_conf = "sudo touch %s" % yum_proxy_file
+    commands.append(add_proxy_conf)
+
+    if FLAGS.http_proxy:
+        add_http_proxy = "echo -e 'proxy= \"%s\";' | sudo tee -a %s" % (FLAGS.http_proxy, yum_proxy_file)
+        commands.append(add_http_proxy)
+
+    self.RemoteCommand(";".join(commands))
```

Comment:
If there's only going to be one command, may as well execute it directly without a command array. But ok either way.